app/views/foreman_webhooks/webhook_templates/ansible_tower_-_host_in_inventory.erb

```diff
@@ -0,0 +1,18 @@
+<%#
+name: Ansible Tower - Host in Inventory
+description: Create/update a host with associated inventory in Ansible Tower.
+snippet: false
+model: WebhookTemplate
+-%>
+<%#
+  Create URL: https://tower/api/v2/hosts
+  Update/Delete URL: https://tower/api/v2/hosts/$HOST_ID
+-%>
+{
```

Comment:
This requires `payload` as well:
```
<%=
  payload({
    "name": @object.name,
    "description": "Created via Foreman Webhook",
    "inventory": @object.host_param('ansible_tower_inventory_id') || 1,
    "enabled": true,
    "instance_id": "",
    "variables": ""
  }, with_defaults: false) # optionally
-%>
```

Reply:
I am not a huge fan of this, but okay.

The comment above, I think blank (empty) payload should be completely blank. Nothing. Zero. Nada. :-)